bin/server.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const app = express()
+
+const initialize = () => {
+  var pathLogger = function(req, res, next) {
+    console.log(`Request '${req.path}' logged via middleware`)
+    next()
+  }
+
+  app.use(pathLogger)
+
+  app.get('/', function(req, res) {
```

Comment:
how can the reflow library user extend this functionality (like adding RESTful apis and serving assets)